python-sdk/nuscenes/prediction/models/covernet.py

```diff
@@ -65,15 +65,15 @@ def forward(self, image_tensor: torch.Tensor,
         return logits
 
 
-def l1_distance(lattice: torch.Tensor, ground_truth: torch.Tensor) -> torch.Tensor:
+def mean_pointwise_l2_distance(lattice: torch.Tensor, ground_truth: torch.Tensor) -> torch.Tensor:
     """
     Computes the index of the closest trajectory in the lattice as measured by l1 distance.
```

Comment:
Update the comment (l1 -> l2) and mention the mean.

Reply:
I guess we could also keep both losses around to provide more options.

Reply:
They're both equivalent so I'd rather just stick with what's in the paper but the ConstantLatticeLoss makes it easy to plug in different metrics.